pkg/storage/kube/build.go

```diff
@@ -102,8 +102,9 @@ func (s *store) CreateBuild(build *brigade.Build) error {
 		},
 		Type: secretTypeBuild,
 		Data: map[string][]byte{
-			"script":  build.Script,
-			"payload": build.Payload,
+			"script":       build.Script,
+			"brigade.json": build.Deps,
```

Comment:
Nit but I think this should better be `deps` rather than `brigade.json` to be consistent with other fields. Other fields seem to be named after their contents, rather than the file names.

Reply:
I would steer away from `deps`, since we plan to add more things to `brigade.json` in the future (such as environment variables for the worker, for example).

Reply:
@mumoshu I agree the discrepancy looks odd.  I discovered if the file doesn't have an extension of `.json`, node's `require` throws an error attempting to parse.  We could work around this for a file with no extension (e.g., `/etc/brigade/deps`) by renaming or reading/rewriting to a local `*.json` file.  WDYT?

```
 $ k logs brigade-worker-01ds62rc3kctgs3sd0agxg123r
/etc/brigade/..2019_11_08_17_59_55.496562507/deps:2
    "dependencies": {
                  ^

SyntaxError: Unexpected token :
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at addDeps (/home/src/prestart.js:35:16)
error Command failed with exit code 1.
```

Reply:
@radu-matei ah I see.  What should we rename the flag and internal fields as?  Currently `-d`/`--deps-file` and `Deps`...  something connoting a Brigade config file?  `--config`/`Config`?

Reply:
Yeah, I think `--config` would work.